internal/policy/policy.go

```diff
@@ -143,13 +142,116 @@ func LoadState(ctx context.Context, repo *git.Repository, entry *rsl.ReferenceEn
 // LoadCurrentState returns the State corresponding to the repository's current
 // active policy. It verifies the root of trust for the state starting from the
 // initial policy entry in the RSL.
-func LoadCurrentState(ctx context.Context, repo *git.Repository) (*State, error) {
-	latestEntry, _, err := rsl.GetLatestReferenceEntryForRef(repo, PolicyRef)
+func LoadCurrentState(ctx context.Context, repo *git.Repository, ref string) (*State, error) {
+	entry, _, err := rsl.GetLatestReferenceEntryForRef(repo, ref)
 	if err != nil {
 		return nil, err
 	}
 
-	return LoadState(ctx, repo, latestEntry)
+	return LoadStateForEntry(ctx, repo, entry)
+}
+
+// LoadStateForEntry returns the State for a specified RSL reference entry for
+// the policy namespace.
+func LoadStateForEntry(ctx context.Context, repo *git.Repository, entry *rsl.ReferenceEntry) (*State, error) {
```

Comment:
I think this is an incorrect rebase after #308? That one makes this a private function.

Reply:
Fixed